scripts/code.angularjs.org-firebase/functions/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "functions-firebase-code.angularjs.org",
   "description": "Cloud Functions to serve files from gcs to code.angularjs.org",
   "engines": {
-    "node": "8"
+    "node": "10"
```

Comment:
Cloud functions do not yet support v12+

Reply:
@petebacondarwin wow, is there a roadmap? Not supporting v12+ when v14 is out is weird.

Reply:
I couldn't find a roadmap. Just this: https://firebase.google.com/docs/functions/manage-functions#set_nodejs_version

Reply:
And in the online editor in the Google Cloud console, there is a dropdown to select node version and it only contains 8 and 10.

Reply:
They are still trying to get people moved from v8 to v10. It's not clear when v12 will even have beta support 😞